### DISCUSSION TITLE
Will the mech suits ever return?

### DISCUSSION THREAD

**Discussion by Horkas** [tsunamayo/Starship-EVO#4394] [Suggestions]
Titles says it all. 

Will the mech suits ever be brought back into the game? Its actually the primary reason I bought the pre-alpha support copy XD. I loosely remember him talking about it being taken out due to technical limitations of.... something he changed. I really really hope that he brings back the ability to make mechs again!

Even if it ends up being a little janky, its better than not having them at all!

**Comment by JRL101** [comment]
From what i know the DEV has removed it why he re-works it. The previous code was not up to his standards. 

But i also have ideas for implementing this using ["rig re-targeting"](https://www.youtube.com/watch?v=fNgPkuMgWFg) [2](https://www.youtube.com/watch?v=KfYjMtBK4LA) where you have the "ideal animation" that drives the procedual rig attached to the players creation. This can be divided up per limb, having the assigned user created mech parts (leg, arm, head) with re-target them selves to the bones of the "ideal animation" it basically translates the joint rotations from the rig and applies them based on the orientation of each of the mechs joints. say if the knee and hip joint on the players mech, only have one axis of rotation, those joints copy the rotation as close as it can to the "ideal animation" which is invisible + non collidable.
